README.md

```diff
@@ -41,3 +50,83 @@ After configuring the service proxies, you can look toward the top, under User S
 
 You will be able to add the name of the key you added (`my_new_proxy` in this example) so that the users can select it.
 The entry with a star is going to be the default selection if a user does not configure it. You can delete all other entries from here to force a single one.
+
+## Modifying Browser Settings
+
+You can configure the following browser settings in URLDownloader:
+
+- User agent(`user_agent`): This is the value of `User-Agent` in request headers.
+- Window size(`window_size`): The screen resolution of the browser.
+- Request headers (`headers`/`request_headers`): Request headers used to fetch URLs.
+
+Administrators can set the default values for these settings from the URLDownloader configuration page in the **Service Variables** section.
+These settings will be used by every single submissions sent to URLDownloader.
+
+Here is an example of `default_browser_settings [json]` service variable:
+
+```
+# iPhone 11 Pro configuration:
+{
+    "window_size": "375x812",
+    "user_agent": "Mozilla/5.0 (iPhone; CPU iPhone OS 14_6 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.0.3 Mobile/15E148 Safari/604.1",
+    "request_headers":
+    {
+        "Sec-CH-UA-Mobile": "?1"
+    }
+}
+```
+
+Users can also modify these settings per URL submission by submitting an Assemblyline URI file.
+Here is an example:
+
+```
+# Assemblyline URI file 1
+uri: https://sample_webpage.com/
+
+browser_settings:
+    window_size: 1280x720
+    user_agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.163 Safari/537.36
+
+# request headers
+headers:
+    sec-ch-ua-platform: Windows
+```
+
+The values set by the URI file will override the values set by `default_browser_settings`.
+If `headers` value is set in the URI file, its value overwrites the service parameter values set in `default_browser_settings.request_headers`.
+
+Using the above `default_browser_settings` and _Assemblyline URI file 1_ as an example, these will be the settings used by URLDownloader:
+
+```
+uri: https://sample_webpage.com/
+
+# settings from Assemblyline URI file 1
+window_size: 1280x720
+user_agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.163 Safari/537.36
```

Comment:
What happens if a user specify both the `user_agent` key and the `headers.User-Agent` key?
What happens if an administrator specify both the `user_agent` key and the `request_headers.User-Agent` key?
What happens if an administrator specify the `user_agent` key and the user specifies the `headers.User-Agent` key?
What happens if a user specify the `user_agent` key and the administrator specifies the `request_headers.User-Agent` key?